infra/modules/providers/azure/ad-application/variables.tf

```diff
@@ -7,6 +7,11 @@ variable "ad_app_config" {
   }))
 }
 
+variable "instance_count" {
```

Comment:
Why are we making this a configuration element? I think this could add confusion...

Reply:
The feature in this PR prompted me to make an app service output configuration change which required a map that mapped app services to their respective slots. The ad-application relies on this new mapping, however, the map is so dynamic that terraform couldn't infer a count for the ad-application to rely on at plan time ("compile time") and would throw a error. My response to that error  prompted to bring back ad-application's "instance count" variable.

Reply:
Found a simple way to keep "instance count" out of the picture. Thanks for calling this out. Will update shortly!

Reply:
Instance count has been removed. Also, screenshot of staging confi and auth login screen is now included in PR description.